src/renderer/components/CreateChat.js

```diff
@@ -10,9 +10,29 @@ const {
   Button
 } = require('@blueprintjs/core')
 
+const styled = require('styled-components').default
+const StyleVariables = require('./style-variables')
+
 const ContactList = require('./ContactList')
 const NavbarWrapper = require('./NavbarWrapper')
 
+const OvalDeltaButton = styled.button`
+  background-color: ${StyleVariables.colors.deltaPrimaryBg} 
```

Comment:
Nice!

Reply:
We could move that into an own file as soon as we want to reuse that button. It's better than the normal button, but still not very nice. It's time for some designer :)